agent/src/fty_srr_groups.cc

```diff
@@ -255,6 +255,24 @@ static auto initSrrFeatures = []() {
     tmp[F_AI_SETTINGS].m_restart     = true;
     tmp[F_AI_SETTINGS].m_reset       = true;
 
+    tmp[F_TIMEZONE_SETTINGS];
+    tmp[F_TIMEZONE_SETTINGS].m_id          = F_TIMEZONE_SETTINGS;
+    tmp[F_TIMEZONE_SETTINGS].m_name        = F_TIMEZONE_SETTINGS;
+    tmp[F_TIMEZONE_SETTINGS].m_description = TRANSLATE_ME("srr_timezone");
+    tmp[F_TIMEZONE_SETTINGS].m_agent       = CONFIG_AGENT_NAME;
+    tmp[F_TIMEZONE_SETTINGS].m_requiredIn  = {"2.3"};
+    tmp[F_TIMEZONE_SETTINGS].m_restart     = false;
+    tmp[F_TIMEZONE_SETTINGS].m_reset       = false;
+
+    tmp[F_NTP_SETTINGS];
+    tmp[F_NTP_SETTINGS].m_id          = F_NTP_SETTINGS;
+    tmp[F_NTP_SETTINGS].m_name        = F_NTP_SETTINGS;
+    tmp[F_NTP_SETTINGS].m_description = TRANSLATE_ME("srr_ntp");
+    tmp[F_NTP_SETTINGS].m_agent       = CONFIG_AGENT_NAME;
+    tmp[F_NTP_SETTINGS].m_requiredIn  = {"2.3"};
+    tmp[F_NTP_SETTINGS].m_restart     = false;
```

Comment:
this is the point
if m_restart == false... fty-config *must* handle the ntp service start/stop 
else fty-config has only to handle mask/enable states for the ntp service.